data/json/monsters/zed_misc.json

```diff
@@ -232,6 +232,8 @@
     "armor_elec": 1,
     "vision_day": 2,
     "vision_night": 0,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
```

Comment:
Needs a new weakpoint set that doesn't include a head critical.

Reply:
Done